scripts/uni_tag_allocation.py

```diff
@@ -0,0 +1,97 @@
+import os
+import sys
+import datetime
+from kytos.core.db import Mongo
+from collections import defaultdict
+
+def update_database(mongo: Mongo):
+    db = mongo.client[mongo.db_name]
+
+    circuits = db.evcs.find({},
```

Comment:
@Alopalao all `db.evcs.find()` will need to only included non archived circuits. Otherwise, it'll end up including non used tags which in turn would ultimately getting removed from `available_vlans`. I believe we have the same issue on [`vlan_pool.py` script](https://github.com/kytos-ng/topology/blob/master/scripts/vlan_pool.py#L59) that has landed on `master`, so if you could also double check and send a new PR fixing that too that'd be great.

Here's an example:

```
rs0 [direct: primary] napps> db.evcs.find({}, {"uni_a": 1, "uni_z": 1, "active": 1, "archived": 1})
[
  {
    _id: '1e555ae332134a',
    active: false,
    archived: true,
    uni_a: {
      tag: { tag_type: 1, value: 6 },
      interface_id: '00:00:00:00:00:00:00:01:1'
    },
    uni_z: {
      tag: { tag_type: 1, value: 6 },
      interface_id: '00:00:00:00:00:00:00:03:1'
    }
  },
  {
    _id: '1026aec04d0547',
    active: true,
    archived: false,
    uni_a: {
      tag: { tag_type: 1, value: 101 },
      interface_id: '00:00:00:00:00:00:00:01:1'
    },
    uni_z: {
      tag: { tag_type: 1, value: 102 },
      interface_id: '00:00:00:00:00:00:00:03:1'
    }
  }
]
rs0 [direct: primary] napps> 
```

Reply:
❯ CMD=aggregate_uni_tags python scripts/uni_tag_allocation.py
Interfaces that probably need their available_vlans modified.
00:00:00:00:00:00:00:01:1: [6, 101]
00:00:00:00:00:00:00:03:1: [6, 102]

Reply:
No longer including archived EVCs.